Contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to DeliverEase
+
+Welcome to DeliverEase! We are excited to have you contribute to our project. DeliverEase is made using Django (Python) for the backend and HTML/CSS for the frontend. Your contributions can help us improve and enhance the platform, empowering delivery partners and connecting communities even more effectively.
+
+## How to Contribute
+
+To contribute to DeliverEase, please follow these guidelines:
+
+1. Fork the repository to your GitHub account.
+
+2. Create a new branch from the `main` branch. Use a descriptive name for your branch that reflects the nature of your contribution.
+
+3. Make your changes to the codebase or documentation in your branch. Be sure to follow the coding style and conventions used in the project.
+
+4. Write clear and concise commit messages for your changes.
```

Comment:
Add a instruction to attach issue number while committing to track it as part of the Issue.
cc: @Devansh-Singh-Parmar